src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -667,7 +664,6 @@ private JabRefPreferences() {
         defaults.put(DO_NOT_RESOLVE_STRINGS_FOR, FieldName.URL);
         defaults.put(RESOLVE_STRINGS_ALL_FIELDS, Boolean.FALSE);
         defaults.put(NON_WRAPPABLE_FIELDS, "pdf;ps;url;doi;file;isbn;issn");
-        defaults.put(GENERATE_KEYS_AFTER_INSPECTION, Boolean.TRUE);
         defaults.put(WARN_ABOUT_DUPLICATES_IN_INSPECTION, Boolean.TRUE);
```

Comment:
Is there a reason why you removed  `GENERATE_KEYS_AFTER_INSPECTION` ?

Reply:
Yes, the keys are now always generated so there is no need to have preference toggle for it.